README.md

```diff
@@ -230,18 +230,18 @@ more verbose or more debug-y logs.
 
 Otherwise, you can start out with `0` as "you always want to see this",
 `1` as "common logging that you might *possibly* want to turn off", and
-`10` as "I would like to performance-test your log collection stack".
+`10` as "I would like to performance-test your log collection stack."
```

Comment:
Again, the difference between period and comma in the style guides seems so silly.

Reply:
Same comment as above.